lib/views/squadra/registra_squadra.dart

```diff
@@ -13,10 +16,12 @@ class ViewRegistraSquadra extends StatefulWidget {
 
 class _ViewRegistraSquadraState extends State<ViewRegistraSquadra> {
   final _formKey = GlobalKey<FormState>();
+  final DatabaseService _dbService = getIt<DatabaseService>();
```

Comment:
if you use the dbService for just the current user ID, save that instead of the whole service